sys/linux/socket_qipcrtr.txt

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+include <linux/socket.h>
+include <linux/net.h>
+include <linux/qrtr.h>
+
+# QIPCRTR sockets
+
+resource sock_qrtr[sock]
+
+sockaddr_qrtr {
+	sq_family	const[AF_QIPCRTR, int16]
+	sq_node		int32
```

Comment:
It makes sense to give these sq_node/sq_port more precise types. I think they are pretty important for getting more coverage.
First, they have some special values:
https://elixir.bootlin.com/linux/v5.7.3/source/net/qrtr/qrtr.c#L98
https://elixir.bootlin.com/linux/v5.7.3/source/include/uapi/linux/qrtr.h#L8
https://elixir.bootlin.com/linux/v5.7.3/source/include/uapi/linux/qrtr.h#L8

Second, even if a value semantically can be any value from int32, frequently we give them reduces range (e.g. `int32[1:4]`) just to increase chances of these values being equal in 2 syscalls in the same program (which presumably may lead to interesting coverage, e.g. bind and connect to the same port).

So e.g. for node we could do flags with the following values:
```
# 1 stands for qrtr_local_nid
qrtr_nodes = QRTR_NODE_BCAST, 0, 1, 2, 3, 4
```